components/horn-service-kuksa/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+#*******************************************************************************
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#******************************************************************************/
+
+[package]
+name = "horn-service-kuksa"
+version = "0.1.0"
+edition = "2021"
+license.workspace = true
+
+[dependencies]
+async-trait = { workspace = true }
+chrono = { workspace = true }
+clap = { workspace = true }
+horn-proto = { workspace = true }
+kuksa = { git = "https://github.com/eclipse-kuksa/kuksa-databroker.git", branch = "feature/refactor_kuksa_crate", package = "kuksa" }
```

Comment:
This branch already exists for some time and needs some minor rework before getting merged, so I am not sure when it will be merged exactly. However, since it is also needed as dependency for the service-to-signal blueprint, I put it on my ToDo list as well to have an actual Kuksa crate available anytime soon.
 
One alternative, would be to generate a Kuksa client locally, during the build phase of the blueprint from the Kuksa interface definition. But to me it seems more efficient to generate such a client once in the scope of the Kuksa project instead of putting the effort to add it to the local build.